modules/swagger-codegen/src/main/resources/lua/api.mustache

```diff
@@ -8,12 +8,14 @@ local dkjson = require "dkjson"
 local basexx = require "basexx"
 
 -- model import
-local {{{packageName}}}_{{classname}} = require "{{{packageName}}}.api.{{classname}}"
+{{#imports}}
+local {{{packageName}}}_{{import}} = require "{{{packageName}}}.model.{{import}}"
+{{/imports}}
 
-local {{{packageName}}}= {}
-local {{{packageName}}}_mt = {
+local {{{classname}}}= {}
```

Comment:
unrelated style: should add a space before `=`

Reply:
Fixed via 39179a2